examples/js/objects/Sky.js

```diff
@@ -39,14 +39,16 @@ THREE.Sky.SkyShader = {
 		"rayleigh": { value: 1 },
 		"mieCoefficient": { value: 0.005 },
 		"mieDirectionalG": { value: 0.8 },
-		"sunPosition": { value: new THREE.Vector3() }
+		"sunPosition": { value: new THREE.Vector3() },
+		"up": { value: new THREE.Vector3(0, 1, 0) }
```

Comment:
Minor style issue. It should be: `new THREE.Vector3( 0, 1, 0 )`.

Notice the additional spaces before and after the brackets.